### PR TITLE
DPTP-2077 create the ipi-deprovision build cofiguration

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  metadata:
+    name: ipi-deprovision
+    namespace: ci
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ipi-deprovision:latest
```
Comment (hongkailiu) [inline clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml:20]: Is it conflicting to the following?

https://github.com/openshift/release/blob/9d75912f1577b2a4d97195a22b9e565a13c6e47c/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L207-L218

Comment (droslean) [reply clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml:20]: I have another PR opened to remove the build in ci-operator config

Comment (droslean) [reply clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml:20]: https://github.com/openshift/release/pull/17069

File: clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+              curl -O https://bootstrap.pypa.io/get-pip.py && \
+              python3 get-pip.py && \
+              rm -fv get-pip.py && \
+              pip install awscli
+      type: Dockerfile
+    strategy:
+      dockerStrategy:
+        from:
+          kind: DockerImage
+          name: docker.io/centos:8
```
[the remark this answers is not in the document]
Comment (droslean) [reply clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml]: updated

File: clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+              curl -O https://bootstrap.pypa.io/get-pip.py && \
+              python3 get-pip.py && \
+              rm -fv get-pip.py && \
+              pip install awscli
+      type: Dockerfile
+    strategy:
+      dockerStrategy:
+        from:
+          kind: DockerImage
+          name: origin/centos:8
```
Comment (hongkailiu) [inline clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml:62]: I would expect something like this.
```
        from:
          kind: ImageStreamTag
          name: centos:8
          namespace: origin
```
Not sure if `DockerImage` will work without a full path. Have you tested?

**Comment by hongkailiu** [comment]
BTW, why do you want to do BC instead of building in CI?

File: clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml (hunk 3 above)
Comment (droslean) [reply clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml]: I didn't test, I thought It works with a full path. Either, way I just updated it.

**Comment by droslean** [comment]
> BTW, why do you want to do BC instead of building in CI?

We don't need this image to be built in every commit since it is not serving any tool that exists in ci-tools.

File: clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml (hunk 3 above)
Comment (droslean) [reply clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml]: It seems that the strategy was wrong too. Updated.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**